container-search/src/test/java/com/yahoo/search/searchers/test/QueryValidatorTestCase.java

```diff
@@ -33,12 +30,21 @@ public void testValidation() {
         new QueryValidator().search(new Query("?query=mystring:foo"), execution);
 
         try {
-            new QueryValidator().search(new Query("?query=mytensor:foo"), execution);
-            fail("Excpected validation error");
+            new QueryValidator().search(new Query("?query=sddocname%3Aproduct%20lfmModel25KeysV0%3A9%2A%20mytensor1%3A%3E0"), execution);
+            fail("Expected validation error");
         }
         catch (IllegalArgumentException e) {
             // success
-            assertEquals("Cannot search 'mytensor': It is a tensor field", e.getMessage());
+            assertEquals("Cannot search 'mytensor1': It is a tensor field", e.getMessage());
+        }
+
+        try {
+            new QueryValidator().search(new Query("?query=sddocname%3Aproduct%20lfmModel25KeysV0%3A9%2A%20mytensor2%3A%3E0"), execution);
```

Comment:
Consider simplifying query to only contain _mytensor2_.

Reply:
👍

Reply:
done, thanks

Reply:
Reason for complicated query perhaps to verify nothing's lost?